floodwatch/src/js/components/Profile.js

```diff
@@ -159,67 +194,54 @@ export class DemographicContainer extends Component {
   updateStateAndMessages(userData: PersonDemographics) {
     this.setState({
       userData: userData,
-      curStatus: (this.state.curStatus != 'error') ? null : 'error'
+      curStatus: (this.state.curStatus !== 'error') ? null : 'error'
     })
   }
 
   render() {
-    let elems = Filters.filters.map((filter) => {
-      if (filter.name == 'age') {
+    let elems = Filters.filters.map((filter, i) => {
+      if (filter.name === 'age') {
         return <AgeOption updateYear={this.updateYear.bind(this)}
                           handleClick={this.handleClick.bind(this)}
-                          userData={this.state.userData} filter={filter}/>
+                          userData={this.state.userData} filter={filter}
+                          key={i}/>
 
-      } else if (filter.name == 'country') {
+      } else if (filter.name === 'country') {
         return <LocationOption
                           updateLocation={this.updateLocation.bind(this)}
                           handleClick={this.handleClick.bind(this)}
-                          userData={this.state.userData} filter={filter}/>
+                          userData={this.state.userData} filter={filter}
+                          key={i}/>
       } else {
         return <DefaultOption
                           handleClick={this.handleClick.bind(this)}
-                          userData={this.state.userData} filter={filter}/>
+                          userData={this.state.userData} filter={filter}
+                          key={i}/>
       }
 
 
     })
 
-    const displayGroup = (this.state.curStatus == null) ? 'none' : 'block'
-
     return (
-      <Row>
-        <Col xs={12}>
+      <div>
+        <div>
         {elems}
-        </Col>
-        <Col xs={12}>
-        <Button onClick={this.updateUserInfo.bind(this)} id="submit-button">Submit</Button>
-        <ListGroup style={{display: displayGroup, textAlign:'center'}}>
-          { this.state.curStatus == 'success' &&
-            <ListGroupItem bsStyle="success">
-              Successfully saved changes!
-            </ListGroupItem>
-          }
-          { this.state.curStatus == 'error' &&
-            <ListGroupItem bsStyle="danger">
-              Error while trying to save changes. Please check your connection.
-            </ListGroupItem>
-          }
-        </ListGroup>
-        </Col>
-      </Row>
+        </div>
+
+        <div className="profile-page_actions">
+          <Button className="profile-page_actions_submit" bsSize="large" bsStyle="primary" onClick={this.updateUserInfo.bind(this)} id="submit-button">Save</Button>
```

Comment:
No, <Button /> is a react-bootstrap component and take care of this. I also was surprised that react components can use HTML5 tags as name.